plugins/dell-kestrel/fu-dell-kestrel-hid-device.c

```diff
@@ -379,11 +362,28 @@ fu_dell_kestrel_hid_device_write_firmware(FuDellKestrelHidDevice *self,
 		g_debug("wait %u ms for dock to finish the chunk", chunk_delay);
 		fu_device_sleep(FU_DEVICE(self), chunk_delay);
 
-		/* ensure the chunk has been acknowledged */
-		if (!fu_dell_kestrel_hid_device_verify_chunk_result(self, i, error))
+		/* get device response for the chunk in transaction */
+		if (!fu_dell_kestrel_hid_device_verify_chunk_result(self, &resp, error))
 			return FALSE;
 
-		fu_progress_step_done(progress);
+		/* verify the device response */
+		message = g_strdup_printf("dock response %u to chunk[%u]: %s",
+					  resp,
+					  i,
+					  fu_dell_kestrel_hid_ec_chunk_response_to_string(resp));
+		g_debug("%s", message);
```

Comment:
i don't get what `message` brings to the table. Can you do

    g_debug("dock response %u to chunk[%u]: %s",
					  resp,
					  i,
					  fu_dell_kestrel_hid_ec_chunk_response_to_string(resp));

Reply:
the message will be reused in the failure case, since the text is very likely the same so brought it up. Is it ideal?